README.md

```diff
@@ -38,6 +39,18 @@ There are two main ways to go about this.
 * Compare performance and resource utilization of generated HDL with pure Verilog and Vivado HLS
 * Compare code size with Verilog, Python, C++ and Vivado HLS as a index of development productivity
 
+#### spi2wb
+- **Developer** Fabien Marteau
+- **Repository** [https://github.com/Martoni/spi2wb](https://github.com/Martoni/spi2wb)
+- **Versions** 1.3, 1.2, 1.1, 1.0
+- **Sbt Dependency** com.github.hplp" %% "spi2wb" % "1.3"
```

Comment:
This seems wrong: @chick 
1. the `com.github.hplp` prefix is copied from the `aes` peripheral
2. I don't think `spi2wb ` is published anywhere so it's not really a `maven`, more of a `local` **Type**.